Dockerfile

```diff
@@ -36,9 +28,81 @@ ADD transmission/runUpdates.sh /etc/transmission-daemon/startPortUpdates.sh
 ADD transmission/down.sh /etc/transmission-daemon/stop.sh
 ADD runOpenVpn.sh /etc/openvpn/start.sh
 
+ENV PIA_USERNAME=**None** \
+    PIA_PASSWORD=**None** \
```

Comment:
I also set these to a default bogus value. You can remove these as well if you'd like.